cnc_ctrl_v1/Settings.h

```diff
@@ -42,6 +42,7 @@ typedef struct {  // I think this is about ~128 bytes in size if I counted corre
   float sledCG;
   byte kinematicsType;
   float rotationDiskRadius;
+  float chainSagCorrection;
```

Comment:
I thought that struct members were stored in the order declared.  Although it appears I may be wrong about that:

https://stackoverflow.com/questions/281045/do-class-struct-members-always-get-created-in-memory-in-the-order-they-were-decl

Still, at the risk of being pedantic, I think this should be down after zPropWeightV.  It doesn't appear to matter but it will make it a bit more logical for us in the future $37 coming after $36.

Reply:
Sounds good to me!